config/registry-config.yaml

```diff
@@ -0,0 +1,41 @@
+# https://docs.docker.com/registry/configuration/
+
+version: 0.1
+log:
+  level: debug
+  fields:
+    service: registry
+storage:
+  cache:
+    blobdescriptor: inmemory
+  filesystem:
+    rootdirectory: /mnt/docker0storage/registry
+http:
+  addr: 443:5000 # https://docs.docker.com/registry/configuration/#letsencrypt
+  host: docker.cdot.systems
+  tls:
+    certificate: /path/to/x509/public
+    key: /path/to/x509/private
+    letsencrypt:
+      cachefile: /path/to/cache-file
```

Comment:
This section seems wrong.  Do we need to do any SSL stuff in here, since nginx is handling it?  I doubt it.